### PR TITLE
Allow the in-memory cache TTL to be specified as a configuration parameter

### PR DESCRIPTION
**Pull request by jkodumal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +73,8 @@ To set up LDR in this mode, provide a redis host and port, and supply a Redis ke
         prefix = "ld:spree:test"
         apiKey = "SPREE_TEST_API_KEY"
 
+You can also configure an in-memory cache for the relay to use so that connections do not always hit redis. To do this, set the `ttl` parameter in your `redis` configuration section to a number (in milliseconds).
```
Comment (drichelson) [inline README.md]: I think we should have the in-memory cache's default TTL be > 0. Anyone running LD-Relay for scale reasons will likely want it enabled.

Comment (jkodumal) [inline README.md]: Good idea-- addressed in latest commit.

File: ld-relay.go [hunk 2]
```diff
@@ -47,6 +49,7 @@ type Config struct {
 	Redis struct {
 		Host string
 		Port int
+		Ttl  *int
```
Comment (jkodumal) [inline ld-relay.go]: Mmm... true. Suggestions?

Comment (drichelson) [inline ld-relay.go]: but if it is consistent with how it is named in the RedisFeatureStore we should either leave it or change it in both places

Comment (drichelson) [inline ld-relay.go]: InMemoryTtl?

**Comment by drichelson** [comment]
closing, as this is not quite the right approach

**Comment by drichelson** [comment]
Sorry. I accidentally closed this.

**Pull request merged**